plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/product-attributes-block-editor.spec.js

```diff
@@ -373,6 +373,11 @@ test(
 			await expect(
 				async () => {
 					await page.reload();
+					await expect(
+						page.locator(
+							`[aria-label="Block: Product attributes"]`
+						)
+					).toBeVisible();
```

Comment:
Without this change, the test was constantly failing against the Pressable website. With it, it passed every time.

Reply:
Done in [8fd64ef](https://github.com/woocommerce/woocommerce/pull/50960/commits/8fd64efe4db7e09667ac848af3b43ee0c0a826f8). Please let me know if more details are needed